python/py-cli-helpers/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-cli-helpers
-version             1.2.1
+version             2.1.0
```

Comment:
this version doesn't list PY27 anymore as supported... please check whether it still works and if not restrict it to the latest supported version for the py27 subport.